lib/bolt/project_migrate/inventory.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'bolt/project_migrate/base'
+
+module Bolt
+  class ProjectMigrate
+    class Inventory < Base
+      def migrate(inventory_file, backup_dir)
+        @outputter.print_message "Migrating inventory\n\n"
```

Comment:
Hm, I'm not actually sure we should print everything we try to do if we don't need to do it, mostly because it's a lot to read (and admittedly because in my terminal `\n` seems to be expanding weirdly). For example, this is the output if there's nothing to do:
```
lucy ~/githubs/bolt/project ((0da2c5b...)) $ be bolt project migrate
Migrating inventory

  → Bolt inventory is already on the latest version. Nothing to do.


Migrating project configuration

  → Found project configuration file

    /home/lucy/githubs/bolt/project/bolt-project.yaml. Nothing to do.


Migrating project modules

  → Project is already using module management workflow. Nothing to do.


Project successfully migrated
```
Which seems like a lot of scanning just to be like "Ok it didn't do anything".

Reply:
Ya, I definitely think cutting down on the messages is warranted. :)